Client/src/Pages/Monitors/Home/monitorData.jsx

```diff
@@ -32,21 +32,30 @@ export const buildData = (monitors, isAdmin, navigate) => {
 
   data.rows = monitors.map((monitor, idx) => {
     let uptimePercentage = "";
+    let percentageColor = theme.palette.success.main; 
+
     if (monitor.uptimePercentage !== undefined) {
       uptimePercentage =
         monitor.uptimePercentage === 0
-          ? "0" 
-          : (monitor.uptimePercentage * 100).toFixed(2); 
-    }
+          ? "0"
+          : (monitor.uptimePercentage * 100).toFixed(2);
+
+          if (monitor.uptimePercentage < 0.25) {
+            percentageColor = theme.palette.error.text;
+          } else if (monitor.uptimePercentage >= 0.25 && monitor.uptimePercentage < 0.5) {
+            percentageColor = theme.palette.warning.main;
+          } else if (monitor.uptimePercentage >= 0.5 && monitor.uptimePercentage < 0.75) {
+            percentageColor = theme.palette.warning.light;
+          } else if (monitor.uptimePercentage >= 0.75 && monitor.uptimePercentage <= 1) {
+            percentageColor = theme.palette.success.main;
+          }
+        }
```

Comment:
What about using ternary operators instead of an if/else ladder?

```
let percentageColor = 
  monitor.uptimePercentage < 0.25 ? theme.palette.error.text :
  monitor.uptimePercentage < 0.5 ? theme.palette.warning.main :
  monitor.uptimePercentage < 0.75 ? theme.palette.warning.light :
  theme.palette.success.main;
  ```
  
seems more readable to me :thinking: 

You could also do something similar with an immediately invoked anonymous function with early returns

```
let percentageColor = (() => {
  if (monitor.uptimePercentage < 0.25) return theme.palette.error.text;
  if (monitor.uptimePercentage < 0.5) return theme.palette.warning.main;
  if (monitor.uptimePercentage < 0.75) return theme.palette.warning.light;
  return theme.palette.success.main;
})();
```

Reply:
Yup either of those options work for me. I will go ahead and implement one of them. Thank you invaluable input. I really appreciate it!